pyglmnet/utils.py

```diff
@@ -72,7 +73,20 @@ def log_likelihood(y, yhat, distr):
 
 def tikhonov_from_prior(PriorCov, n_samples):
     """Given a prior covariance matrix, returns a Tikhonov matrix"""
+    """Given a prior covariance matrix, returns a Tikhonov matrix"""
     [U, S, V] = np.linalg.svd(PriorCov, full_matrices=False)
-    Tau = np.dot(np.diag(1. / np.sqrt(S)), U)
-    Tau = 1. / np.sqrt(np.float(n_samples)) * Tau / Tau.max()
+
+    S_ratio = S / S.max()
+    threshold = 0.0001
```

Comment:
dumb question, why do we have a `threshold` here and why is it hard coded? can you point me to the equations somewhere?